include/spirv/unified1/spirv.py

```diff
@@ -428,6 +428,7 @@
         'HlslCounterBufferGOOGLE' : 5634,
         'HlslSemanticGOOGLE' : 5635,
         'UserSemantic' : 5635,
+        'UserTypeGOOGLE' : 5635,
```

Comment:
This is using the wrong number.  But I'm about to submit a PR that fixes this (a side effect of using the header generation tool)

Reply:
Thanks